introduction/deprecated_features/CAPMC_Deprecation_Notice.md

```diff
@@ -1,47 +1,41 @@
 # CAPMC Deprecation Notice
 
-## CAPMC enters final life cycle before deletion
+## CAPMC enters final life cycle before removal
 
-PCS v1 has been released to production and indicated the beginning of the final life cycle for CAPMC. CAPMC will
-continue to co-exist with PCS until its final deletion. CAPMC is slated to be permanently removed from the system
-beginning with the first release of software in 2024. Please begin to migrate to PCS.
+CSM 1.5 is the last release that will contain the CAPMC service. It has been
+replaced by PCS. CAPMC continues to co-exist with PCS to assist in transition to
+PCS. CAPMC is slated to be permanently removed from the system beginning with
+CSM 1.6.
 
-### Deprecated Features in CSM 1.0
+### Status of CAPMC in CSM 1.6
 
-Many CAPMC v1 REST API and CLI features are being deprecated as part of CSM version 1.0; Full removal of the
-following deprecated CAPMC features will happen in CSM version 1.3. Further development of CAPMC service or CLI
-has stopped. CAPMC has entered end-of-life but will still be generally available. CAPMC is going to be replaced
-with the Power Control Service (PCS) in a future release. The current API/CLI portfolio for CAPMC are being pruned
-to better align with the future direction of PCS. More information about PCS and the CAPMC transition will be
-released as part of subsequent CSM releases.
+CAPMC was deprecated in CSM 1.5 and intended to be removed from CSM 1.6.  The
+decision was made to not remove it in CSM 1.6 so that existing dependencies
+that have not yet transitioned to PCS do not break.  Support for CAPMC however,
+is removed starting in CSM 1.6.  We encourage everyone to transition to PCS as
+soon as possible.
 
-The API endpoints that remain un-deprecated will remain supported until their 'phased transition' into PCS (e.g.
-Power Capping is not 'deprecated' and will be supported in PCS; As PCS is developed, CAPMC's Power Capping and
-PCS's Power Capping will both function, eventually callers of the CAPMC power capping API/CLI will need to will
-need transition to call PCS as the API will be different.)
+### Deprecated Features in CSM 1.5
+
+CAPMC has been end-of-life since the CSM 1.3 release. The remaining APIs and CLI commands
```

Comment:
This paragraph can probably just be combined with the reworded paragraph from above, since the only new information in it is that CAPMC has been end-of-life since CSM 1.3.

Reply:
That section was the section added for 1.5.  The section I'm adding is the update for 1.6.  Might be nice to have the historical context when customers read this?  Let me know if you agree or not, can update it if not.

Reply:
I don't feel strongly about it, except that we should definitely remove the part that says that it is removed in CSM 1.6